pages/search/index.tsx

```diff
@@ -161,18 +161,21 @@ export default function Search({
             )}
             <Pagination
               id="results-pagination"
-              mt="xl"
+              mt="xxl"
+              mb="l"
               initialPage={searchParams.page}
               currentPage={searchParams.page}
               pageCount={Math.ceil(totalResults / RESULTS_PER_PAGE)}
               onPageChange={handlePageChange}
             />
           </>
+        ) : /**
+         * TODO: The logic and copy for different scenarios will need to be added when
+         * filters are implemented
+         */
+        isLoading ? (
+          <SkeletonLoader showImage={false} />
         ) : (
-          /**
```

Comment:
This is not related to VQA but a small bug fix I noticed in passing. Fixes an issue where there was no loading state after searching from a no results page. Let me know if you'd like me to move this to a separate PR.